src/Qdrant.Client/Grpc/Expression.cs

```diff
@@ -0,0 +1,126 @@
+namespace Qdrant.Client.Grpc
+{
+    /// <summary>
+    /// To construct a formula for score boosting.
+    /// </summary>
+    public partial class Expression
```

Comment:
Looks like there are no tests for any of these, at least to have examples to point folks to in how to use them.

Reply:
We'll have example snippets at qdrant.tech/documentation and api.qdrant.tech/

See https://github.com/qdrant/landing_page/pull/1506

Reply:
Nice 👍 

Something that may save the snippets and source code drifting in future is to pull the snippets from source code.